doc/source/tutorial/stats/rv_infrastructure.md

```diff
@@ -0,0 +1,772 @@
+X.sample(rng=rng), X.sample(rng=1)
+```
+
+Second, the argument `shape` replaces argument `size`.
+
+```{code-cell} ipython3
+dist.rvs(size=(3, 4))
+```
+
+```{code-cell} ipython3
+X.sample(shape=(3, 4))
+```
+
+Besides the difference in name, there is a difference in behavior when array shape parameters are involved. Previously, the shape of distribution parameter arrays had to be included in `size`.
+
+```{code-cell} ipython3
+dist.rvs(size=(3, 4, 2), loc=[0, 1]).shape  # `loc` has shape (2,)
+```
+
+Now, the shape of the parameter arrays is considered to be a property of the random variable object itself. Specifying the shape of array shape parameters would be redundant, so it is not included when specifying the `shape` of the sample.
+
+```{code-cell} ipython3
+Y = stats.Normal(mu = [0, 1])
+Y.sample(shape=(3, 4)).shape  # the sample has shape (3, 4); each element is of shape (2,)
+```
+
+```{code-cell} ipython3
+
+```
+
+#### Probability Mass (AKA "Confidence") Intervals
+
++++
+
+The old infrastructure offered an `interval` method which, by default, would return a symmetric (about the median) interval containing a specified percentage of the distribution's probability mass.
+
+```{code-cell} ipython3
+a = 0.95
+dist.interval(confidence=a)
+```
+
+Now, call the inverse CDF and complementary CDF methods with the desired probability mass.
+
+```{code-cell} ipython3
+p = 1 - a
+X.icdf(p/2), X.iccdf(p/2)
+```
+
+#### Likelihood Function
+
++++
+
+The old infrastructure offered a function to compute the *n*egative *l*og-*l*ikelihood *f*unction, erroneously named `nnlf` (instead of `nllf`). It accepts the parameters of the distribution as a tuple and observations as an array.
+
+```{code-cell} ipython3
+mu = 0
+sigma = 1
+data = stats.norm.rvs(size=100, loc=mu, scale=sigma)
+stats.norm.nnlf((mu, sigma), data)
+```
+
+Now, simply compute the negative [log-likehood function](https://en.wikipedia.org/wiki/Likelihood_function) according to its mathematical definition.
+
+```{code-cell} ipython3
+X = stats.Normal(mu=mu, sigma=sigma)
+-X.logpdf(data).sum()
+```
+
+#### Expected Values
+
++++
+
+The `expect` method of the old infrastructure estimates a definite integral of an arbitrary function weighted by the PDF of the distribution. For instance, the fourth moment of the distribution is given by:
+
+```{code-cell} ipython3
+def f(x): return x**4
+stats.norm.expect(f, lb=-np.inf, ub=np.inf)
+```
+
+This provides little added convencience over what the source code does: use `scipy.integrate.quad` to perform the integration numerically.
+
+```{code-cell} ipython3
+from scipy import integrate
+def f(x): return x**4 * stats.norm.pdf(x)
+integrate.quad(f, a=-np.inf, b=np.inf)  # integral estimate, estimate of the error
+```
+
+Of course, this can just as easily be done with the new infrastructure.
+
+```{code-cell} ipython3
+def f(x): return x**4 * X.pdf(x)
+integrate.quad(f, a=-np.inf, b=np.inf)  # integral estimate, estimate of the error
+```
```

Comment:
It wasn't removed. Users are welcome to use `expect` from the old interface. It hasn't been a priority for me to add to the new interface because be really can't do much to improve upon the existing numerical integration features (or those we have recently added). Faced with the choice between spending time recreating these interfaces vs adding value, I chose the latter. Also, sometimes users seem to expect more from `expect` than it can reasonably provide (e.g. gh-8928, gh-9259, gh-14347). If `expect` were not an option, perhaps users will learn to perform numerical integration with the general tools. At their simplest, the general tools are not any more complex to use, but they give you a lot more control when things don't "just work".